source/blog/2016-08-18-running-kubernetes-in-containers-on-atomic.html.md

```diff
@@ -0,0 +1,179 @@
+
+On the master:
+
+```
+# docker run -d \
+    --net=host \
+    --pid=host \
+    --privileged \
+    --restart="unless-stopped" \
+    --name kube_kubelet_$(date | md5sum | cut -c-5) \
+    -v /sys:/sys:rw \
+    -v /var/run:/var/run:rw \
+    -v /run:/run:rw \
+    -v /var/lib/docker:/var/lib/docker:rw \
+    -v /var/lib/kubelet:/var/lib/kubelet:shared \
+    -v /var/log/containers:/var/log/containers:rw \
+    gcr.io/google_containers/hyperkube-amd64:$(curl -sSL "https://storage.googleapis.com/kubernetes-release/release/stable.txt") \
+    /hyperkube kubelet \
+      --allow-privileged \
+      --api-servers=http://localhost:8080 \
+      --config=/etc/kubernetes/manifests-multi \
+      --cluster-dns=10.0.0.10 \
+      --cluster-domain=cluster.local \
+      --hostname-override=${MASTER_IP} \
+      --v=2
+```
+
+On the worker:
+
+```
+# export WORKER_IP=YOUR-WORKER-IP
+
+# docker run -d \
+    --net=host \
+    --pid=host \
+    --privileged \
+    --restart="unless-stopped" \
+    --name kube_kubelet_$(date | md5sum | cut -c-5) \
+    -v /sys:/sys:rw \
+    -v /var/run:/var/run:rw \
+    -v /run:/run:rw \
+    -v /var/lib/docker:/var/lib/docker:rw \
+    -v /var/lib/kubelet:/var/lib/kubelet:shared \
+    -v /var/log/containers:/var/log/containers:rw \
+    gcr.io/google_containers/hyperkube-amd64:$(curl -sSL "https://storage.googleapis.com/kubernetes-release/release/stable.txt") \
+    /hyperkube kubelet \
+      --allow-privileged \
+      --api-servers=http://${MASTER_IP}:8080 \
+      --cluster-dns=10.0.0.10 \
+      --cluster-domain=cluster.local \
+      --hostname-override=${WORKER_IP} \
+      --v=2
+
+# docker run -d \
+    --net=host \
+    --privileged \
+    --name kube_proxy_$(date | md5sum | cut -c-5) \
+    --restart="unless-stopped" \
+    gcr.io/google_containers/hyperkube-amd64:$(curl -sSL "https://storage.googleapis.com/kubernetes-release/release/stable.txt") \
+    /hyperkube proxy \
+        --master=http://${MASTER_IP}:8080 \
+        --v=2
+```
+
+### get current kubectl
+
+I usually test things out from the master node, so I'll download the newest stable kubectl binary to there:
+
+```
+# curl -sSL https://storage.googleapis.com/kubernetes-release/release/$(curl -sSL "https://storage.googleapis.com/kubernetes-release/release/stable.txt")/bin/linux/amd64/kubectl > /usr/local/bin/kubectl
+
+# chmod +x /usr/local/bin/kubectl
+```
+
+### test it
+
+It takes a few minutes for all the containers to get up and running. Once they are, you can start running kubernetes apps. I typically test with the [guestbookgo atomicapp](https://github.com/projectatomic/nulecule-library/tree/master/guestbookgo-atomicapp):
```

Comment:
I would add a `kubectl get nodes` here to help users following along to sanity check their work so far.